pos_show_clock/readme/CONFIGURE.rst

```diff
@@ -0,0 +1 @@
+To configure this module, you need to do nothing.
```

Comment:
Useless file if no configuration is needed.

Reply:
> Lignt code review. No test. Same comment as chienandalou regarding V16 comments.
> 
> Lgtm otherwise. Thanks ! Coule you add a printscreen in the description ?

Can we keep the js comment of v16? Because i made this module to version 16 first to better understand OWL v2